core-bundle/src/Monolog/ContaoTableProcessor.php

```diff
@@ -102,7 +102,7 @@ private function updateSource(ContaoContext $context, Request|null $request = nu
 
     private function updateUri(ContaoContext $context, Request|null $request = null): void
     {
-        if (!$request) {
+        if (!$request || null !== $context->getUri()) {
```

Comment:
This change is needed for the `CspReportListener` to set the URL from the report.